changelog.md

```diff
@@ -21,7 +39,7 @@
 * Drop support for Python 3.5
 
 ## Minor Changes
-* Update Readme 
+* Update Readme
 * Remove runhistory from TAE (#663)
```

Comment:
I would recommend using "Separated evaluation from get next challengers in intensification." instead of "remove runhistory from TAE", as described by the PR to show what actually happened in this PR

Reply:
This was change written fro `0.13.0` that I didn't, do but I will change it anywho